packages/core/components/DirectoryTree/DirectoryTreeNodeHeader.tsx

```diff
@@ -47,7 +47,10 @@ export default React.memo(function DirectoryTreeNodeHeader(props: DirectoryTreeN
     const { collapsed, error, fileSet, isLeaf, isFocused, loading, onClick, title } = props;
 
     const [isContextMenuActive, setContextMenuActive] = React.useState(false);
-
+    // Shorten folder title length to approximately one line
+    // Some overflow is acceptable as long as the folder doesn't wrap to more than two lines
+    const isLongHeader = title.length > 110;
```

Comment:
Yes!! however isn't this reliant on the screen size? could we calculate this using the screen size if so to be a bit more rad?

Reply:
Tried this out in [5b0798e](https://github.com/AllenInstitute/biofile-finder/pull/520/commits/5b0798eed5ff68a39f55fe78340da4d15c6d6b2b), currently available [on staging](https://staging.bff.allencell.org/app?c=file_name%3A0.4%2CKind%3A0.2%2CType%3A0.25%2Cfile_size%3A0.15&group=Notes&filter=%7B%22name%22%3A%22uploaded%22%2C%22value%22%3A%22RANGE%282024-05-09T07%3A00%3A00.000Z%2C2025-05-10T06%3A59%3A59.558Z%29%22%2C%22type%22%3A%22default%22%7D&source=%7B%22name%22%3A%22AICS+FMS%22%7D&sort=%7B%22annotationName%22%3A%22uploaded%22%2C%22order%22%3A%22DESC%22%7D&showNulls=true) -- tested by changing window width